package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "SOURCE_MAP=true PORT=4000 vue-cli-service serve ./src/main.js",
+    "dev": "SET PORT=4000 && vue-cli-service serve ./src/main.js",
```

Comment:
no need to define with `SET`, and removing `SOURCE_MAP=true`
just declare it in your `.env.develomplent.local` file.

please revert it

Reply:
oh, again collision, forgot about changed locally package.json
so sad it's impossible make local version of package and use it always, like .env
will be fixed

Reply:
still not resolved